src/components/vm/consoles/vnc.jsx

```diff
@@ -419,6 +450,16 @@ export class VncActive extends React.Component {
             />
         );
 
+        const vnc_tls = qemu_conf.vnc_tls == "1";
+
+        function get_failure_reason() {
+            if (state.connection_failure)
+                return state.connection_failure;
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2038-7e41038d-20250806-115005-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)